manifests/config.pp

```diff
@@ -40,6 +40,12 @@
           group   => 'root',
           content => template('consul/consul.systemd.erb'),
         }
+        exec { 'systemctl daemon-reload':
+          path        => ['/bin','/usr/bin','/sbin','/usr/sbin'],
```

Comment:
I don't know if its in the same place in every RH variant, or system that uses systemd.  Figured these were pretty likely paths.  Happy to change it.